src/app/management-center/catalogs/catalogs.component.scss

```diff
@@ -1,3 +1,5 @@
 .app-catalogs {
+    display: flex;
```

Comment:
@danieltolmon do you still need this ?

Reply:
test it ;)

Reply:
Yeap , I dont need it any more. Chaooo!!